### ISSUE TITLE
Put full data insight into atom feed

### ISSUE THREAD

**Issue by larsyencken** [owid/owid-grapher#3759]

**Comment by rakyi** [comment]
@edomt could you check whether [this](http://staging-site-add-content-to-di-atom-feed/atom-data-insights.xml) would work for Mailchimp?

**Comment by edomt** [comment]
Thank you @rakyi! I just tested it and it works well in Mailchimp. The one issue is that images within `<content type="HTML">` only have a relative path:

```html
<img src="/images/published/coal-consumption-by-country-terawatt-hours-twh-mobile%402x.png" alt="" width="1080" height="1080"/>
```

And so Mailchimp can't load them. Could we have an absolute path here?

**Comment by rakyi** [comment]
Added the absolute path now, but not sure if you'll be able to make Mailchimp use Tailscale. We can do the good old test in prod, if not. 😄 The images show up correctly in my local RSS reader.

**Comment by larsyencken** [comment]
I think it's likely you'll have to test in prod. On the upside, nobody will be affected if you iterate there on a feed that is not yet advertised or used.

**Comment by edomt** [comment]
Yes, we'll have to bring it to prod, as Mailchimp needs to be able to access the images directly! Let me know when you have it there (not urgent!)

**Comment by rakyi** [comment]
It is now in prod. However, I modified the existing atom feed, as originally suggested, which is already advertised. I think it makes sense to have a single nicer feed, but I can bring back the old one and put the new one on a different URL if we want.

**Comment by edomt** [comment]
Thanks! I think that overwriting the existing one should be fine 🆗
